<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ajouter</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tri">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>752</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="ajouter_carte">
     <property name="styleSheet">
      <string notr="true">#ajouter_carte{
background:rgba(255,0,0,100%);
}</string>
     </property>
     <attribute name="title">
      <string>ajouter_carte</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>NUMPASSEPORT</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>points :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>offres :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>210</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>VALIDITE:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>ID_FID :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>250</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>RESET</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SUMBIT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pic">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>110</y>
        <width>231</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Avg/Desktop/activations-carte-fidelite.jpg</pixmap>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>310</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>434</x>
        <y>130</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="modifier">
     <attribute name="title">
      <string>modifier</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>ID- FID :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>VALIDITE :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>points :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>offres :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>240</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>NUMPASSEPORT</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>200</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_13">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>240</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>RESET</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SUMBIT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="afficher">
     <attribute name="title">
      <string>afficher</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>80</y>
        <width>321</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>tri</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="supprimer_carte">
     <property name="styleSheet">
      <string notr="true">#supprimer_carte{
background:rgba(255,0,0,100%);
}</string>
     </property>
     <attribute name="title">
      <string>supprimer_carte</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>96</x>
        <y>70</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>id_carte:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SUMBIT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>RESET</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="scanner">
     <property name="styleSheet">
      <string notr="true">#scanner{
background:rgba(255,0,0,100%);
}</string>
     </property>
     <attribute name="title">
      <string>scanner code barre</string>
     </attribute>
     <widget class="QLabel" name="id_client">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>id_client:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>code barre</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>120</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SUMBIT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>RESET</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
